sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-webapp/README.md

```diff
@@ -1,74 +1,78 @@
-   - Select **Delegated permissions**, and then click **ActivityFeed.Read**
-   - Select the **Add permissions** button
-   - Click **Grant Permissions...** and Yes when prompted.
+### Configure web api
+1. Search for and select Azure Active Directory.
+2. Under Manage, select **App registrations** -> **New registration**.![Protal manage](docs/image-protal-manage.png "Protal manage")
+3. The registered application name is filled into **arm**, select **Accounts in this organizational directory only**, click **register** button.![Register a web api](docs/image-register-a-web-api.png "Register a web api")
+4. Under **arm** application, select **Certificates & secrets** -> **new client secret**, expires select **Never**, click **add** button.(Remember to save the secrets here and use it later)![Creat secrets](docs/image-creat-secrets-api.png "Creat secrets")
+5. Under **arm** application, select **Expose an API** -> **Add a scope**, Use the default **Application ID URI**, click **Save and continue** button.![Expose apis](docs/image-set-app-id-url.png "Expose apis")
+6. After step five,the page will refresh again.Then set the **Scope name** to *Files.read*.(This scope will be used in Application.yml)![Add a app id url](docs/image-add-a-scope.png "Add a app id url")
+7. Expose scopes in web API registration.![Expose api](docs/image-expose-api.png "Expose api")
+
+See [Register app or web api], [Expose scopes in web api registration] for more information about the Web API.
+
+### Configure web app
+1. Under Manage In the same tenant, select **App registrations** -> **New registration**.![Protal manage](docs/image-protal-manage.png "Protal manage")
+2. The registered application name is filled into **webapp**, select **Accounts in this organizational directory only**, click the **register** button.![Register a web app](docs/image-register-a-web-app.png "Register a web app")
+3. Under **webapp** application, select **Certificates & secrets** -> **new client secret**, expires select **Never**, click the **add** button.(Remember to save the secrets here and use them later)![Creat secrets](docs/image-creat-secrets-app.png "Creat secrets")
+4. Under **webapp** application, select **Authentication** -> **Add a platform**, select **web** platform, redirect urls set to `http://localhost:8080/login/oauth2/code/azure`, check the Access Token and ID Token checkboxes, click **configure** button.![Add a platfform](docs/image-add-a-platfform.png "Add a platfform")
+5. Under **webapp** application, select **Authentication** -> **Add URI**, you need to add redirect URIs of `http://localhost:8080/login/oauth2/code/arm`. ![App add url](docs/image-app-add-url.png "App add url")
+6. Under **webapp** application, select **API permissions** -> **Add a permission**, select **Microsoft Graph**. Next, search **Calendar.Read** via select Permissions, check the check box,click **add permissions** button.![Api permission](docs/image-api-permissions.png "Api permission")
```

Comment:
Not in integration test, we use scopes like this:
![image](https://user-images.githubusercontent.com/13167207/103070203-fb4a7200-45fb-11eb-8c05-931d8be44f94.png)

Could you please use the same scope in sample?